examples/ibm-power/variables.tf

```diff
@@ -1,33 +1,94 @@
-// Service / Account
+# Service // Account
 variable "ibm_cloud_api_key" {
   description = "API Key"
   type        = string
   default     = "<key>"
 }
 variable "region" {
-  description = "Reigon of Service"
+  description = "Region of Service"
   type        = string
   default     = "<e.g dal>"
 }
 variable "zone" {
   description = "Zone of Service"
   type        = string
-  default     = "<e.g 12>"
+  default     = "<e.g dal12>"
 }
-variable "cloud_instance_id" {
-  description = "Cloud Instance ID of Service"
-  type        = string
-  default     = "<cid>"
+
+# Workspace
+variable "workspace_name" {
+  description = "Workspace Name"
+  type = string
+  default = "<name>"
+}
+variable "datacenter" {
+  description = "Datacenter Region"
```

Comment:
Provide a link to the docs where a user could find the list of data centers to choose from

Reply:
Made changes